src/service.rs

```diff
@@ -145,7 +145,13 @@ impl ServiceControl for NodeServiceManager {
     }
 
     fn is_port_free(&self, port: u16) -> bool {
-        TcpListener::bind(("127.0.0.1", port)).is_ok()
+        if cfg!(feature = "tcp") {
+            TcpListener::bind(("127.0.0.1", port)).is_ok()
+        } else if cfg!(feature = "quic") {
+            UdpSocket::bind(("127.0.0.1", port)).is_ok()
+        } else {
+            false
```

Comment:
Compiling this crate without either `tcp` and `quic` feature doesn't really make sense?

Reply:
aye, the compiler complained that it _could_ still. I'm not sure of the best default strat here 🤔